### ISSUE TITLE
Can't install packages due to libpng-dev causing errors

### ISSUE THREAD

**Issue by furyrulez** [retr0rangepi/RetrOrangePi#48]
![IMG_20190408_192527](https://user-images.githubusercontent.com/49411961/55746221-7c816000-5a39-11e9-9b05-fccbabe6cd0d.jpg)
I updated both Retrorangepi and the Retropie-setup script... but I still get this message when I try to install dosbox-sdl2, lr-caprice, eduke32... Is this normal? (I updated Mame2003 from source, installed Tyrquake, and some other stuff with no problem, btw).

**Comment by retr0rangepi** [comment]
fixed 
https://github.com/retr0rangepi/RetroPie-Setup/commit/6966ff55fe4cfa1a9b9714df22a792c71d8ec8f9

update retropie-setup script again

**Comment by furyrulez** [comment]
I updated the retropie-setup script again, then I tried to install "eduke32" (Duke Nukem 3D port), and it still can't be installed. It stops during the process, and on the screen it writes "failed building sdlmusic.o from source/duke3d/src/sdlmusic.cpp" before exiting.
Then I tried "super flappy birds", and it can't be installed. 
Plus: Not sure if related to the update, but today I got 2 errors while browsing systems in emulationstation with the background music ON, they said something about an error with the audio mixer, audio card... I can't remember exactly.
